Jellyfin.Plugin.Tvdb/Providers/TvdbSeriesProvider.cs

```diff
@@ -61,6 +62,8 @@ public async Task<IEnumerable<RemoteSearchResult>> GetSearchResults(SeriesInfo s
         }
 
         /// <inheritdoc />
+        [SuppressMessage("Microsoft.Maintainability", "CA1725: Parameter names should match base declaration", Justification = "Requires a MediaBrowser.Controller.Providers interface change to fix")]
+        // TODO: Rename itemId to either info as declared in the interface or rename parameter in the MediaBrowser.Controller.Providers interface
```

Comment:
Just refactoring this function should be fine (vscode is usually helpful by just pressing F2 and renaming the variable)

Reply:
I am on that Jetbrains fair, so it is just as easy! (was just not sure if crobibero had anything in mind when he used those names :P )